src/AppInstallerCLICore/Workflows/PortableFlow.cpp

```diff
@@ -239,7 +263,10 @@ namespace AppInstaller::CLI::Workflow
             AppInstaller::Filesystem::AppendExtension(fileName, ".exe");
             AppInstaller::Filesystem::AppendExtension(commandAlias, ".exe");
 
-            const std::filesystem::path& targetFullPath = targetInstallDirectory / fileName;
+            const std::filesystem::path& targetFullPath = isSymlinkCreationSupported ?
+                targetInstallDirectory / fileName :
+                targetInstallDirectory / commandAlias;
+
             entries.emplace_back(std::move(PortableFileEntry::CreateFileEntry(installerPath, targetFullPath, {})));
             entries.emplace_back(std::move(PortableFileEntry::CreateSymlinkEntry(symlinkDirectory / commandAlias, targetFullPath)));
```

Comment:
Do not create symlink entry here if symlink creation not supported. Use the same flag in portable entry in archive portable case above.